app/javascript/components/payment-error-button.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="payment-error-button">
+    <modal name="apology-modal">
+      <div class="invoice__apology-modal-text">
+        <h1>Error en el pago</h1>
+        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+      </div>
+    </modal>
+    <div class="invoice__error-btn ">
+        <button
+          type="button" v-if="showApologyButton"
+          class="btn"
+          @click="show"
+        >
+          ¿No funciona?
+        </button>
+    </div>
+  </div>
+</template>
+
+<script>
+
+import { mapState, mapActions, mapGetters } from 'vuex';
```

Comment:
'mapState' is defined but never used                                       no-unused-vars<br>'mapActions' is defined but never used                                     no-unused-vars<br>'mapGetters' is defined but never used                                     no-unused-vars